packages/f-checkout/test/specs/component/f-checkout.component.spec.js

```diff
@@ -28,10 +28,47 @@ describe('f-checkout component tests', () => {
             postcode: 'AR51 1AA'
         };
 
+        const usernote = 'No mushrooms! Doorbell number 2 :)'
+
         // Act
         CheckoutComponent.submitCheckoutForm(addressInfo);
-        CheckoutComponent.inputUserNote('No mushrooms!');
-        CheckoutComponent.selectFulfillmentTime('As soon as possible');
+        CheckoutComponent.inputUserNote(usernote);
+        CheckoutComponent.selectOrderTime('As soon as possible');
         CheckoutComponent.submit();
     });
+
+    it('should display times in ascending order, with default text "As soon as possible" showing first', () => {
+        // Assert
+        expect(CheckoutComponent.isOrderTimeDropdownDisplayed()).toBe(true);
+        // expect(CheckoutComponent.getOrderTimeOptionText(0)).toBe('As soon as possible');
+        // expect(CheckoutComponent.getOrderTimeOptionText(1)).toBe('Monday 00:15');
+        // expect(CheckoutComponent.getOrderTimeOptionText(2)).toBe('Monday 00:30');
+    });
+
+    it('should allow the user to click through different order times', () => {
+        // Act
+        CheckoutComponent.orderTimeDropdownOptions().forEach((element) => {
+            element.click();
+        });
+
+        CheckoutComponent.selectOrderTime(' As soon as possible ');
+    });
+
+    it('should prevent user from writing a note of over 200 characters', () => {
+        // Arrange 
+        const usernote = 'A'
+        const maxUserNote = usernote.repeat(300);
+
+        console.log('HEYYY', CheckoutComponent.inputUserNoteLength(maxUserNote))
+
+        // Assert
+        expect(CheckoutComponent.userNoteMaxCharacterCount()).toEqual('200');
+        expect(CheckoutComponent.inputUserNoteLength(maxUserNote)).toEqual(200)
+    });
+
+    it.skip('should enable a user to submit without adding a note', () => {
+        // Act
+        browser.refresh(); 
```

Comment:
What's this refresh for?

Reply:
can be removed - will do that now!

Reply:
Don't think this test can be finished until the button leads to a new page, so we can grab the new url and show that it submits